ic-helper.ps1

```diff
@@ -66,22 +66,35 @@ function Find-GitRepositories {
         [string]$command,
         [switch]$force
     )
-    $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+
+ # Check if the path exists
+    if (-not (Test-Path $path)) {
+        Write-Error "The specified path '$path' does not exist."
+         return
+    }
+
+    try {
+        $subdirs = Get-ChildItem -Path $path -Directory -Recurse
+    }
+    catch {
+        Write-Error "An error occurred while retrieving subdirectories: $($_.Exception.Message)"
```

Comment:
Simple message stating that the specified directory does not exist would be sufficient. It is also a good idea to add absolute path to this directory in the message.